module/VuFind/src/VuFind/SMS/Clickatell.php

```diff
@@ -79,14 +79,14 @@ public function text($provider, $to, $from, $message)
         try {
             $result = $this->client->setMethod('GET')->setUri($url)->send();
         } catch (\Exception $e) {
-            throw new MailException($e->getMessage());
+            throw new MailException($e->getMessage(), MailException::ERROR_UNKNOWN);
         }
         $response = $result->isSuccess() ? trim($result->getBody()) : '';
         if (empty($response)) {
-            throw new MailException('Problem sending text.');
+            throw new MailException('Problem sending text.', MailException::ERROR_UNKNOWN);
```

Comment:
Isn't this change going to cause the error message to say "Problem sending email" instead of "Problem sending text"? That might cause confusion.